nav2_amcl/src/amcl_node.cpp

```diff
@@ -1066,6 +1087,10 @@ AmclNode::initParameters()
   get_parameter("initial_pose.y", initial_pose_y_);
   get_parameter("initial_pose.z", initial_pose_z_);
   get_parameter("initial_pose.yaw", initial_pose_yaw_);
+  get_parameter("initial_cov.x", initial_cov_x_);
+  get_parameter("initial_cov.y", initial_cov_y_);
+  get_parameter("initial_cov.z", initial_cov_z_);
```

Comment:
I don't think that the z covariance is useful. I doubt that AMCL uses it.

Reply:
Removed initial_cov.z